src/panels/lovelace/editor/config-elements/hui-shopping-list-editor.ts

```diff
@@ -0,0 +1,87 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import { struct } from "../../common/structs/struct";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+import "@polymer/paper-toggle-button/paper-toggle-button";
```

Comment:
You only use paper-input, so that should be imported, the rest can go.

Reply:
yeah, cleaning up now :)